beerocks/bwl/dummy/mon_wlan_hal_dummy.cpp

```diff
@@ -6,7 +6,7 @@
  * See LICENSE file for more details.
  */
 
-#include "mon_wlan_hal_dwpal.h"
+#include "mon_wlan_hal_dummy.h"
 
 #include <beerocks/bcl/beerocks_utils.h>
```

Comment:
Are all these includes relevant for the dummy implementation?

Reply:
They might not be, but then again I expect the dummy implementation to be deleted once we bring up DWPAL + hwsim on Linux, no point in having 2 stub implementations.

Reply:
Might make sense to add a comment or readme that this will be removed. Then again, it will be removed so quickly that it's not worth the bother :-)